best-practices-cel/disallow-empty-ingress-host/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: disallow-empty-ingress-host
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: ../disallow-empty-ingress-host.yaml
+    - assert:
+        file: policy-ready.yaml
```

Comment:
This chainsaw test assumes that the policy is already in Enforce mode whereas the other chainsaw tests in the library patch the policy to set it to Enforce from Audit

Reply:
I have made the necessary changes